sage-spkg-info.py

```diff
@@ -0,0 +1,353 @@
+#!/usr/bin/env sage-bootstrap-python
+"""
+Python rewrite of `build/bin/sage-spkg-info` with suggestions applied:
+  - Rewrote actually most of the original script because 
+  - Utilizes Python's logging (instead of print) for debug/info.
+    - Re-use/initialize logging via sage_bootstrap if available.
+  - Call `sage-print-system-package-command` instead of writing sudo lines.
+
+NOTE: Many functions presume Sage's bootstrap helpers are in PATH (e.g. `sage-package`).
+      Could try to Adjust `SAGE_ROOT`, PATH, or run from within bootstrap like within the original script.
+
+Could try the following invoking method:
+cd ~/passagemath (SAGE_ROOT)
+export SAGE_ROOT="$PWD"
+export PATH="$PWD/build/bin:$PATH"
+python3 build/bin/sage-spkg-info.py 4ti2 --output-rst --log-level INFO
+"""
+
+from __future__ import annotations
+from functools import cached_property
+import sys
+from pathlib import Path
```

Comment:
also in this file, please replace use of pathlib. Needs to run with python2 because it is part of sage_bootstrap